tests/unit/test_systemd_notify.py

```
@@ -0,0 +1,271 @@
+        module, fake_notify, FakeNotification = (
+            self._load_task_module_with_mock_cysystemd()
+        )
+        assert (
+            module._sd_notify is not None
+        ), "_sd_notify should be defined when cysystemd is available"
+        module._sd_notify("READY=1")
+        fake_notify.assert_called_once_with(FakeNotification.READY)
+
+    def test_unknown_kind_does_not_call_raw(self):
+        """Unknown _kind strings must not trigger any notify call."""
+        module, fake_notify, FakeNotification = (
```

Ruff (0.15.9)

[warning] Unpacked variable `FakeNotification` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)